site/app/controllers/admin/SqlToolboxController.php

```diff
@@ -24,7 +24,19 @@ class SqlToolboxController extends AbstractController {
      * @Route("/courses/{_semester}/{_course}/sql_toolbox", methods={"GET"})
      */
     public function showToolbox(): WebResponse {
-        return new WebResponse(SqlToolboxView::class, 'showToolbox');
+        $this->core->getCourseDB()->query("SELECT * FROM information_schema.columns WHERE table_schema='public'");
```

Comment:
We can sort the table names as part of our SQL query:

```suggestion
        $this->core->getCourseDB()->query("SELECT * FROM information_schema.columns WHERE table_schema='public' ORDER BY table_name ASC, ordinal_position ASC");
```

We also want to sort on `ordinal_position` as there's no guarantee that column order in this table is stable / guaranteed compared to what you'd see by looking at table metadata in `psql` or some other software.